src/components/05_pages/Content/Content.js

```diff
@@ -175,9 +195,38 @@ class Content extends Component {
           </Button>
         </div>
 
+        <div className={styles.filters}>
+          {this.props.actions && (
+            <FormControl className={styles.formControl}>
```

Comment:
This class adds some margin to the element which makes it look a bit strange.
<img width="219" alt="screen shot 2018-06-14 at 15 09 34" src="https://user-images.githubusercontent.com/1845495/41399614-f8e28704-6fe4-11e8-9aa1-f351b7361456.png">

Reply:
I fixed it, 
![screen shot 2018-06-15 at 17 47 56](https://user-images.githubusercontent.com/29678/41479803-43de361a-70c4-11e8-801e-13511e165f8d.jpg)